app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%if@recipes.empty?%>
+<h1 class="text-important font-important text-center">The list is empty</h1>
+<%else%>
+<ul class="flex flex-col gap-5">
+  <% @recipes.each do |recipe|%>
+  <li>
+    <a href="/users/<%= @user.id%>/recipes/<%=recipe.id%>" class="grid grid-cols-1 sm:grid-cols-3 p-3 bg-gradient-to-r hover:from-orange-400 hover:to-yellow-500 min-h-[130px] text-start hover:shadow-[10px_9px_#00a195] border-b border-black cursor-pointer rounded-md">
+      <div class="col-span-1 relative flex flex-col text-center">
+        <h2 class="font-important text-important text-center m-auto"><%= recipe.name %></h2>
+        <%= button_to "Remove", user_recipe_path(current_user,recipe),class: "button underline hover:text-red-600", method: :delete %>
```

Comment:
Kindly ensure, when a user deletes an item, they should be redirected back to the page displaying the list of items, rather than the individual item's show page.